### ISSUE TITLE
System tray transparency

### ISSUE THREAD

**Issue by mrlinuxfish** [jaagr/polybar#382]
I have tried multiple settings to get the tray to be the same transparency as the rest of the bar I am using, but the system tray is always either full transparency or no transparency depending on the `tray-transparent` setting.

I am also getting an error `error: Invalid _XEMBED_INFO for window 0x0e016d3` whenever there is anything in the tray.

Additionally, how can I put the tray somewhere more useful, say between the temp. and the clock?

**Comment by jaagr** [comment]
That's a limitation of the tray, as described on the wiki.

The error message is caused by an invalid tray client and is not related to polybar.

In what way would that be more useful? Try the offset parameters. See the wiki for more details.

**Comment by kingkobra87** [comment]
No idea if this is possible or not, but would a hacky workaround be possible? Perhaps if there are two layers - the systray itself, and another background behind that? The systray could then be set to fully transparent, and the adjustment could be on the fake background. Just throwing the idea out there.

**Comment by jaagr** [comment]
@kingkobra87 That's almost how it's done now, except that it just blends with the root pixmap and not the bar window. Additional compositing is needed, but for me it's a relatively low priority right now.

**Comment by patrick96** [comment]
#595 will probably fix this, once it is done.
